client/services/FeedbackGeneratorService.js

```diff
@@ -54,11 +54,12 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     return {
-      getFeedback: function(prompt, codeEvalResult) {
+      getFeedback: function(
+          prompt, codeEvalResult, rawCodeLineIndexes) {
```

Comment:
Or, rather, I should have noticed this earlier, but I'd like to see some tests (or updates to tests) that verify that this works as intended in a future PR.